docker/build_dagmc.sh

```diff
@@ -29,4 +28,3 @@ cmake ../src -DMOAB_DIR=${moab_install_dir} \
 make -j${jobs}
 make install
 cd
-#rm -rf ${dagmc_build_dir}
```

Comment:
The dagmc_build_dir would normally be deleted after building DAGMC, but we need to keep it in this specific case in order to run the tests.

Reply:
My same question above applies here too- do you think it should be left in as a comment w/ an explanation or removed?

Reply:
@ljacobson64 

agreed with @kkiesling if this is comment without explanation it should not be part of the build.
I'd be happy to add them back with a line of explanation. 

when you mean run the tests, do you mean run the test locally ? 
Because we run the test in CI without those...

Reply:
I think this line can be deleted.

To answer @bam241's question: I do mean for running the tests in CI. The CI runs `make test` in the DAGMC build directory, so the DAGMC build directory must exist in order to run the tests. See here: https://github.com/bam241/DAGMC/blob/fix_docker/docker/travis.tests.sh#L28